packages/amplication-server/src/core/pluginInstallation/dto/UpdatePluginInstallationArgs.ts

```diff
@@ -5,5 +5,5 @@ import { PluginInstallationUpdateInput } from "./PluginInstallationUpdateInput";
 @ArgsType()
 export class UpdatePluginInstallationArgs extends UpdateBlockArgs {
   @Field(() => PluginInstallationUpdateInput, { nullable: false })
-  data!: PluginInstallationUpdateInput;
+  declare data: PluginInstallationUpdateInput;
```

Comment:
why do we need to add `declare`?

Reply:
Because we are redefining the `data` property changing the type from the base object.